tests/vppoutputencode.cpp

```diff
@@ -44,15 +44,17 @@ TranscodeParams::TranscodeParams()
 
 bool VppOutputEncode::init(const char* outputFileName, uint32_t /*fourcc*/, int width, int height)
 {
+    EncodeParams encParam;
 
-    if(!width || !height)
+    if (!width || !height)
         if (!guessResolution(outputFileName, width, height))
             return false;
 
     m_fourcc = VA_FOURCC('N', 'V', '1', '2');
     m_width = width;
     m_height = height;
-    m_output.reset(EncodeOutput::create(outputFileName, m_width, m_height));
+    m_output.reset(EncodeOutput::create(outputFileName, m_width, m_height,
+                                        encParam.codec.c_str()));
```

Comment:
The constructor will assign AVC.  Do you have command line examples where this code is exercised by the vpp?